### PR TITLE
Authentication: Phone-number-verification-success-UI

### PR DESCRIPTION
**Pull request by modibboyusuph**
Authentication: Phone-number-verification-success-UI #8

### CODE REVIEW

**Comment by ingressive4good** [comment]
@modibboyusuph there is a conflict on your PR kindly resolve it so that your PR can be merged

**Comment by ingressive4good** [comment]
kindly resolve your conflict

**Comment by ingressive4good** [comment]
Hello @modibboyusuph please fix the conflict on your PR

File: src/App.jsx [hunk 1]
```diff
@@ -1,54 +1,17 @@
 import React from "react";
-import EmailVerify from "./component/email/EmailVerify";
-import Home from "./molecules/Home";
-import EmptyWishlistViewProducts from "./pages/EmptyWishlistViewProducts";
-import WishlistViewProducts from "./pages/WishlistViewProducts";
-import AddNewProductPage1 from './AddNewProduct/pageOne/AddNewProductPage1'
-import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
+
```
Comment (t-bello7) [inline src/App.jsx:2]: @modibboyusuph  You did not have to delete the previous code to fix the conflict.

File: src/App.jsx [hunk 2]
```diff
@@ -1,54 +1,17 @@
...
-              <div className="p-6">
-              <h1 className="text-3xl font-bold text-red-600">Hello world!</h1>
-              <p>/wishlist (view products in wishlist)</p>
-              <p>/add-to-wishlist (add products to wishlist)</p>
-              </div>
-          } exact></Route>
-          <Route path="/wishlist" element={<WishlistViewProducts />}></Route>
-          <Route  path="/add-to-wishlist" element={<EmptyWishlistViewProducts />}></Route>
-      </Routes>
+
```
Comment (t-bello7) [inline src/App.jsx:12]: Kindly add these codes back

File: src/App.jsx [hunk 3]
```diff
@@ -1,54 +1,17 @@
...
+      <PhoneNumberVerificationSuccessUI/>
+
```
Comment (t-bello7) [inline src/App.jsx:14]: Great job designing the Phone Number Verification Success UI. However, add your components to the routes following the same convention as the codes you deleted. You can read the react-router dom documentation if you have any issue

**Review by t-bello7 — APPROVED** [review]

**Comment by dprincecoder** [comment]
Do this to resolve the conflicts

file can’t be merged right now
Why:
You have a conflicts to resolve.
proTip: To resolve, first if you forked the repo then sync it with i4g repo so everything is up to date
Second: checkout to the develop branch and do git pull.
third: checkout to a new branch called resolve conflicts, then if you can copy the code you already did in old branch that conflicts then do else you can code afresh, when done, checkout to this branch that has conflicts then merge this new branch called resolve conflicts you created earlier then you can now see all conflicting files to resolve it.
To make it simple in vscode open all files showing the conflicts then hovering on them will ask you to either accept incoming changes, keep both Changes or compare changes, here click on accept incoming changes after all this, you will see a button that say mark as resolve click it, do so in all conflicting files, you should be good, then commit your changes

**Comment by t-bello7** [comment]
You're almost there @modibboyusuph. You just need to make a few changes.
Screenshot from 2022-10-11 15-12-52
- Fix the merge conflict by updating your code with the develop branch.
- The route to your components comes up blank can you fix this
- Also make the routes pathname to your components all lowercase

**Comment by ingressive4good** [comment]
@modibboyusuph please fix this merge conflict so that your PR can be merged

**Pull request closed without merging**